GNUmakefile

```diff
@@ -17,6 +17,9 @@ EXTRA_CLEAN_TARGETS	= ancillary-clean
 
 -include Makefile
 
+# Define release-related variables
+include $(top_srcdir)/mk/release.mk
+
```

Comment:
If autogen.sh and configure haven't been run first, won't this mean $(top_srcdir) will be undefined and this will try to include /mk/release.mk?

Reply:
whoops good catch -- i can default top_srcdir to .